src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -59,6 +59,8 @@ Baggage is a standard HTTP header with URI encoded key-value pairs.
 For the propagation of DSC, SDKs first read the DSC from the baggage header of incoming requests/messages.
 To propagate DSC to downstream SDKs/services, we create a baggage header (or modify an existing one) through HTTP request instrumentation.
 
+The `baggage` header should only be attached to an outgoing request, if the request's URL matches at least one entry of the [`tracePropagationTargets`](/sdk/performance/#tracepropagationtargets) SDK option (or its fallback default behaviour).
```

Comment:
```suggestion
The `baggage` header should only be attached to an outgoing request if the request's URL matches at least one entry of the [`tracePropagationTargets`](/sdk/performance/#tracepropagationtargets) SDK option.
```

All of our options have default values, and the default behavior is/should be just following the default value, so I don't think saying so is necessary here.

Reply:
Good point